hydra-node/json-schemas/logs.yaml

```diff
@@ -1489,9 +1492,7 @@ definitions:
         snapshot:
           $ref: "api.yaml#/components/schemas/Snapshot"
         signatories:
-          type: array
-          items:
-            $ref: "api.yaml#/components/schemas/Party"
+          type: object
```

Comment:
REVIEW: this is a `Map Party (Signature (Snapshot tx))`
does it makes sense to break this type down into a more granular spec?

Reply:
See comment above